contribution/mobile/pl/item/lore.json

```diff
@@ -55,7 +55,7 @@
       "Gdy światło odbija się od niego, kolor tworzy kilka przyjemnych dla oka odcieni. Ten przedmiot posiada kilka ulepszeń, takich jak zaawansowana SI, interejs neuronowy i łączność z mediami społecznościowymi w wirtualnej rzeczywistości.",
       "Ta błyskotka należała kiedyś do słynnego przywódcy gangu, który poprowadził atak na pierwszą rebelię Sztucznej Inteligencji. Zaginęła po jego śmierci, ale teraz w jakiś sposób znalazła się w Twoim posiadaniu.",
       "Relikt przeszłości, który znalazł drogę z powrotem do cyberprzestrzeni. Nierozpoznawalny, runiczny symbol został wygrawerowany na jego boku. Jego wiek szacuje się na erę pre-cyberpunkową.",
-      "Wspaniały, egzemplarz ${tierName} ekwipunku. Chodzą słuchy, że był w posiadaniu pewnego wirtualnego, czerwonowłosego celebryty."
+      "Wspaniały egzemplarz legendarnego ekwipunku. Chodzą słuchy, że był w posiadaniu pewnego wirtualnego, czerwonowłosego celebryty."
```

Comment:
i dont think this will work congruently since you changed the ${tiername} bit. instead, on the translation file, change the translation for "legendary" to the more accurate one

Reply:
Yeah, i had a problem with tiers, because in polish you have diffrent ways of saying for example "legendary" based on type (masculine/feminine) of the item and i didn't knew how to make it work. I managed to do it by replacing the tier name to for example "of legendary rarity" - i will correct this in next update :)